docs/reference_guide.md

```diff
@@ -1063,6 +1063,20 @@ Attaching 1 probe...
 Unsafe uprobe in the middle of the instruction: /bin/bash:main+1
 ```
 
+Using --unsafe option you can also place uprobes on arbitrary addresses.
+This might come in handy when the binary is stripped.
+```
+$ echo 'int main(){return 0;}' | gcc -xc -o bin -
+$ nm bin | grep main
+...
+0000000000001119 T main
+...
+$ strip bin
+# bpftrace --unsafe -e 'uprobe:bin:0x1119 { printf("main called\n"); }'
+Attaching 1 probe...
+Unsafe uprobe:bin:4377 not matching any symbol at address 0x1119
```

Comment:
Update output?

Reply:
Yup. I forgot about that. Now it's changed.